cal/cal/templates/home_logged_in.html

```diff
@@ -41,55 +42,59 @@
     </li>
   </ul>
   {% if request.user.profile.authed %}
-    <div class="jumbotron">
-
-      <div ng-controller="CalendarCtrl as $ctrl" class="cal-calendar">
 
-        <div ui-calendar="uiConfig.calendar" calendar="panalytics" class="calendar" ng-model="$ctrl.eventSources"></div> 
-        <div class="btn-group">
-          <button type="button"
-            class="btn btn-default dropdown-toggle"
-            data-toggle="dropdown"
-            aria-haspopup="true"
-            aria-expanded="false">
-            Enable / disable calendars
-            <span class="caret"></span>
-          </button>
-          <ul class="dropdown-menu">
-            <li ng-repeat="cal in calendars">
-              <a href="#"> 
-                <input type="checkbox" ng-model="cal.enabled" ng-click="$ctrl.refresh('panalytics')">
-                {% verbatim %} {{ cal.calendar_id }} {% endverbatim %}
-              </a>
-            </li>
-          </ul>
-        </div>
-      </div>
+  <div ng-controller="CalendarCtrl as $ctrl" class="cal-calendar col-md-8">
+    <!-- Enable / disable calendars button -->
+    <div class="btn-group">
+      <button type="button"
+        class="btn btn-default dropdown-toggle"
+        data-toggle="dropdown"
+        aria-haspopup="true"
+        aria-expanded="false">
+        Select Calendars
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li ng-repeat="cal in calendars">
+          <a href="#">
+            <input type="checkbox" ng-model="cal.enabled" ng-click="$ctrl.refresh('panalytics')">
+            {% verbatim %} {{ cal.calendar_id }} {% endverbatim %}
```

Comment:
Now that the summary code is merged in, you can do cal.summary here and reference your issue (: